### PR TITLE
Improve root build & watch scripts

### PR DESCRIPTION
**Pull request by msujew**
The new `build` and `watch` scripts automatically build the TypeScript code as well as the langium-vscode extension. Running both watcher processes in parallel required two terminals previously. Now it runs in the same terminal.

### CODE REVIEW

**Review by kaisalmen — APPROVED** [review]
Tested with Gitpod. Looks good! LGTM

**Comment by kaisalmen** [comment]
Gitpod changes work fine! Thank you for fixing it. 🙂 LGTM 🎉

File: packages/langium-vscode/esbuild.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ require('esbuild').build({
     loader: { '.ts': 'ts' },
     external: ['vscode'], // the vscode-module is created on-the-fly and must be excluded.
     platform: 'node', // VSCode extensions run in a node process
-    sourcemap: true,
+    sourcemap: !minify,
```
Comment (spoenemann) [inline packages/langium-vscode/esbuild.js:23]: Should we use different settings for dev and production?

Comment (msujew) [reply packages/langium-vscode/esbuild.js:23]: Since `dev` is effectively without minification, I don't really see a reason right now.

**Pull request merged**